### DISCUSSION TITLE
Is it possible to use this with an iphone chat.db backup?

### DISCUSSION THREAD

**Discussion by ConnerWithAnE** [ReagentX/imessage-exporter#255] [Q&A] [answered]
From my looking around I assume that things like attachments would not be saveable but would messages be on the table? I can access the database and look through it but I dont understand it all too well as there are a lot of guids floating around.

**Comment by ReagentX** [comment, marked as answer]
Yes, `imessage-exporter` supports iOS backups, as noted in the README.
